tests/unit/commands/_utils/test_options.py

```diff
@@ -527,3 +528,20 @@ def test_skip_without_hook_package(self):
             skip_prepare_infra_callback(ctx_mock, param_mock, True)
 
         self.assertEqual(str(ex.exception), "Missing option --hook-name")
+
+
+class TestNextCommandSuggestions(TestCase):
+    def test_generate_next_command_recommendation(self):
+        listOfTuples = [
+            ("Validate SAM template", "sam validate"),
+            ("Test Function in the Cloud", "sam sync --stack-name {{stack-name}} --watch"),
+            ("Deploy", "sam deploy --guided"),
+        ]
+        output = generate_next_command_recommendation(listOfTuples)
+        expectedOutput = """
+Commands you can use next
+=========================
+[*] Validate SAM template: sam validate
+[*] Test Function in the Cloud: sam sync --stack-name {{stack-name}} --watch
+[*] Deploy: sam deploy --guided"""
```

Comment:
Does this show up correctly in the terminal?

There's no final newline character like before (previously the final `"""` was on a separate line).

Reply:
Yes shows exactly the way it used to show before like shown below -  with the `"""` being in the separate line it would leave a new line, so moved it up.

```
Commands you can use next
=========================
[*] Validate SAM template: sam validate
[*] Test Function in the Cloud: sam sync --stack-name {{stack-name}} --watch
[*] Deploy: sam deploy --guided
```

Reply:
yes, done!